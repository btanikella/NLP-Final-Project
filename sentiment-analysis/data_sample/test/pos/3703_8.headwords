masterpiece
delivers
set
loses
great
written
say
establish
