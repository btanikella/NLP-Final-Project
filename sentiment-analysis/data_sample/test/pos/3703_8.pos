NNP  VBZ RB DT NN IN DT IN PRP$ NN NNS
RB  PRP VBZ DT RB JJ NN  IN IN DT JJ NN
DT JJ NN VBZ VBN RP CD JJ NNS  CD IN PRP VBG DT JJ  DT CD VBG NNS WDT VBP CC VBP IN JJ NN
DT NN RB VBZ DT JJ NN  CC PRP VBZ DT RB JJ NN IN DT JJ NN IN PRP$ JJ NN -LRB- WDT VBZ DT NN IN PRP -RRB-
DT NN VBZ JJ  CC IN DT PRP VBP NN TO DT VBG NN  NNP NNP
RB  DT NNS VBP RB RB VBN CC RB NN : RB  RB PRP VBP VBN IN PRP  PRP MD VB PRP$ NNS IN PRP RB VBG NN
PRP MD RB VB PRP VBD DT NN IN IN PDT DT NN PRP VBD -LRB- IN NNP NNP -RRB-  CC RB IN IN DT NN PRP RB VBZ IN VBG DT JJ NN IN WP VBZ WRB PRP VBP IN DT NN IN NNS POS NNS
RB  NNP MD VB DT NN IN JJ NNS IN NNP NNP  IN PRP VBZ DT RB JJ NN IN JJ NN NN
