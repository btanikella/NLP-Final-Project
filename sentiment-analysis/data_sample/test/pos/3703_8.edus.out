Sentence #1 (14 tokens):
"Caribe" is not a masterpiece in any of its storytelling aspects.
[Text=`` CharacterOffsetBegin=4 CharacterOffsetEnd=5 PartOfSpeech=``] [Text=Caribe CharacterOffsetBegin=5 CharacterOffsetEnd=11 PartOfSpeech=NNP] [Text='' CharacterOffsetBegin=11 CharacterOffsetEnd=12 PartOfSpeech=''] [Text=is CharacterOffsetBegin=13 CharacterOffsetEnd=15 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=16 CharacterOffsetEnd=19 PartOfSpeech=RB] [Text=a CharacterOffsetBegin=20 CharacterOffsetEnd=21 PartOfSpeech=DT] [Text=masterpiece CharacterOffsetBegin=22 CharacterOffsetEnd=33 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=34 CharacterOffsetEnd=36 PartOfSpeech=IN] [Text=any CharacterOffsetBegin=37 CharacterOffsetEnd=40 PartOfSpeech=DT] [Text=of CharacterOffsetBegin=41 CharacterOffsetEnd=43 PartOfSpeech=IN] [Text=its CharacterOffsetBegin=44 CharacterOffsetEnd=47 PartOfSpeech=PRP$] [Text=storytelling CharacterOffsetBegin=48 CharacterOffsetEnd=60 PartOfSpeech=NN] [Text=aspects CharacterOffsetBegin=61 CharacterOffsetEnd=68 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=68 CharacterOffsetEnd=69 PartOfSpeech=.] 
root(ROOT-0, masterpiece-7)
punct(masterpiece-7, ``-1)
nsubj(masterpiece-7, Caribe-2)
punct(masterpiece-7, ''-3)
cop(masterpiece-7, is-4)
neg(masterpiece-7, not-5)
det(masterpiece-7, a-6)
prep(masterpiece-7, in-8)
pobj(in-8, any-9)
prep(any-9, of-10)
poss(aspects-13, its-11)
nn(aspects-13, storytelling-12)
pobj(of-10, aspects-13)
punct(masterpiece-7, .-14)

Sentence #2 (15 tokens):
However, it delivers a perfectly enjoyable story, along with a magnificent cinematography.
[Text=However CharacterOffsetBegin=74 CharacterOffsetEnd=81 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=81 CharacterOffsetEnd=82 PartOfSpeech=,] [Text=it CharacterOffsetBegin=83 CharacterOffsetEnd=85 PartOfSpeech=PRP] [Text=delivers CharacterOffsetBegin=86 CharacterOffsetEnd=94 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=95 CharacterOffsetEnd=96 PartOfSpeech=DT] [Text=perfectly CharacterOffsetBegin=97 CharacterOffsetEnd=106 PartOfSpeech=RB] [Text=enjoyable CharacterOffsetBegin=107 CharacterOffsetEnd=116 PartOfSpeech=JJ] [Text=story CharacterOffsetBegin=117 CharacterOffsetEnd=122 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=122 CharacterOffsetEnd=123 PartOfSpeech=,] [Text=along CharacterOffsetBegin=124 CharacterOffsetEnd=129 PartOfSpeech=IN] [Text=with CharacterOffsetBegin=130 CharacterOffsetEnd=134 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=135 CharacterOffsetEnd=136 PartOfSpeech=DT] [Text=magnificent CharacterOffsetBegin=137 CharacterOffsetEnd=148 PartOfSpeech=JJ] [Text=cinematography CharacterOffsetBegin=149 CharacterOffsetEnd=163 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=163 CharacterOffsetEnd=164 PartOfSpeech=.] 
root(ROOT-0, delivers-4)
advmod(delivers-4, However-1)
punct(delivers-4, ,-2)
nsubj(delivers-4, it-3)
det(story-8, a-5)
advmod(enjoyable-7, perfectly-6)
amod(story-8, enjoyable-7)
dobj(delivers-4, story-8)
punct(delivers-4, ,-9)
prep(delivers-4, along-10)
pcomp(along-10, with-11)
det(cinematography-14, a-12)
amod(cinematography-14, magnificent-13)
pobj(with-11, cinematography-14)
punct(delivers-4, .-15)

Sentence #3 (29 tokens):
The narrative structure is set around two narrative axes, one of them embedding the other, each one having elements that relate and co-exist in excellent harmony.
[Text=The CharacterOffsetBegin=169 CharacterOffsetEnd=172 PartOfSpeech=DT] [Text=narrative CharacterOffsetBegin=173 CharacterOffsetEnd=182 PartOfSpeech=JJ] [Text=structure CharacterOffsetBegin=183 CharacterOffsetEnd=192 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=193 CharacterOffsetEnd=195 PartOfSpeech=VBZ] [Text=set CharacterOffsetBegin=196 CharacterOffsetEnd=199 PartOfSpeech=VBN] [Text=around CharacterOffsetBegin=200 CharacterOffsetEnd=206 PartOfSpeech=RP] [Text=two CharacterOffsetBegin=207 CharacterOffsetEnd=210 PartOfSpeech=CD] [Text=narrative CharacterOffsetBegin=211 CharacterOffsetEnd=220 PartOfSpeech=JJ] [Text=axes CharacterOffsetBegin=221 CharacterOffsetEnd=225 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=225 CharacterOffsetEnd=226 PartOfSpeech=,] [Text=one CharacterOffsetBegin=227 CharacterOffsetEnd=230 PartOfSpeech=CD] [Text=of CharacterOffsetBegin=231 CharacterOffsetEnd=233 PartOfSpeech=IN] [Text=them CharacterOffsetBegin=234 CharacterOffsetEnd=238 PartOfSpeech=PRP] [Text=embedding CharacterOffsetBegin=239 CharacterOffsetEnd=248 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=249 CharacterOffsetEnd=252 PartOfSpeech=DT] [Text=other CharacterOffsetBegin=253 CharacterOffsetEnd=258 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=258 CharacterOffsetEnd=259 PartOfSpeech=,] [Text=each CharacterOffsetBegin=260 CharacterOffsetEnd=264 PartOfSpeech=DT] [Text=one CharacterOffsetBegin=265 CharacterOffsetEnd=268 PartOfSpeech=CD] [Text=having CharacterOffsetBegin=269 CharacterOffsetEnd=275 PartOfSpeech=VBG] [Text=elements CharacterOffsetBegin=276 CharacterOffsetEnd=284 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=285 CharacterOffsetEnd=289 PartOfSpeech=WDT] [Text=relate CharacterOffsetBegin=290 CharacterOffsetEnd=296 PartOfSpeech=VBP] [Text=and CharacterOffsetBegin=297 CharacterOffsetEnd=300 PartOfSpeech=CC] [Text=co-exist CharacterOffsetBegin=301 CharacterOffsetEnd=309 PartOfSpeech=VBP] [Text=in CharacterOffsetBegin=310 CharacterOffsetEnd=312 PartOfSpeech=IN] [Text=excellent CharacterOffsetBegin=313 CharacterOffsetEnd=322 PartOfSpeech=JJ] [Text=harmony CharacterOffsetBegin=323 CharacterOffsetEnd=330 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=330 CharacterOffsetEnd=331 PartOfSpeech=.] 
root(ROOT-0, set-5)
det(structure-3, The-1)
amod(structure-3, narrative-2)
nsubjpass(set-5, structure-3)
auxpass(set-5, is-4)
prt(set-5, around-6)
num(axes-9, two-7)
amod(axes-9, narrative-8)
dobj(set-5, axes-9)
punct(axes-9, ,-10)
appos(axes-9, one-11)
prep(one-11, of-12)
pobj(of-12, them-13)
partmod(one-11, embedding-14)
det(other-16, the-15)
dobj(embedding-14, other-16)
punct(axes-9, ,-17)
det(one-19, each-18)
appos(axes-9, one-19)
partmod(one-19, having-20)
dobj(having-20, elements-21)
nsubj(relate-23, that-22)
rcmod(elements-21, relate-23)
cc(relate-23, and-24)
conj(relate-23, co-exist-25)
prep(co-exist-25, in-26)
amod(harmony-28, excellent-27)
pobj(in-26, harmony-28)
punct(set-5, .-29)

Sentence #4 (32 tokens):
The story never loses the right pacing, and it shows a very good blending of the human figure with its natural environment (which is a character in itself).
[Text=The CharacterOffsetBegin=336 CharacterOffsetEnd=339 PartOfSpeech=DT] [Text=story CharacterOffsetBegin=340 CharacterOffsetEnd=345 PartOfSpeech=NN] [Text=never CharacterOffsetBegin=346 CharacterOffsetEnd=351 PartOfSpeech=RB] [Text=loses CharacterOffsetBegin=352 CharacterOffsetEnd=357 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=358 CharacterOffsetEnd=361 PartOfSpeech=DT] [Text=right CharacterOffsetBegin=362 CharacterOffsetEnd=367 PartOfSpeech=JJ] [Text=pacing CharacterOffsetBegin=368 CharacterOffsetEnd=374 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=374 CharacterOffsetEnd=375 PartOfSpeech=,] [Text=and CharacterOffsetBegin=376 CharacterOffsetEnd=379 PartOfSpeech=CC] [Text=it CharacterOffsetBegin=380 CharacterOffsetEnd=382 PartOfSpeech=PRP] [Text=shows CharacterOffsetBegin=383 CharacterOffsetEnd=388 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=389 CharacterOffsetEnd=390 PartOfSpeech=DT] [Text=very CharacterOffsetBegin=391 CharacterOffsetEnd=395 PartOfSpeech=RB] [Text=good CharacterOffsetBegin=396 CharacterOffsetEnd=400 PartOfSpeech=JJ] [Text=blending CharacterOffsetBegin=401 CharacterOffsetEnd=409 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=410 CharacterOffsetEnd=412 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=413 CharacterOffsetEnd=416 PartOfSpeech=DT] [Text=human CharacterOffsetBegin=417 CharacterOffsetEnd=422 PartOfSpeech=JJ] [Text=figure CharacterOffsetBegin=423 CharacterOffsetEnd=429 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=430 CharacterOffsetEnd=434 PartOfSpeech=IN] [Text=its CharacterOffsetBegin=435 CharacterOffsetEnd=438 PartOfSpeech=PRP$] [Text=natural CharacterOffsetBegin=439 CharacterOffsetEnd=446 PartOfSpeech=JJ] [Text=environment CharacterOffsetBegin=447 CharacterOffsetEnd=458 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=459 CharacterOffsetEnd=460 PartOfSpeech=-LRB-] [Text=which CharacterOffsetBegin=460 CharacterOffsetEnd=465 PartOfSpeech=WDT] [Text=is CharacterOffsetBegin=466 CharacterOffsetEnd=468 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=469 CharacterOffsetEnd=470 PartOfSpeech=DT] [Text=character CharacterOffsetBegin=471 CharacterOffsetEnd=480 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=481 CharacterOffsetEnd=483 PartOfSpeech=IN] [Text=itself CharacterOffsetBegin=484 CharacterOffsetEnd=490 PartOfSpeech=PRP] [Text=-RRB- CharacterOffsetBegin=490 CharacterOffsetEnd=491 PartOfSpeech=-RRB-] [Text=. CharacterOffsetBegin=491 CharacterOffsetEnd=492 PartOfSpeech=.] 
root(ROOT-0, loses-4)
det(story-2, The-1)
nsubj(loses-4, story-2)
neg(loses-4, never-3)
det(pacing-7, the-5)
amod(pacing-7, right-6)
dobj(loses-4, pacing-7)
punct(loses-4, ,-8)
cc(loses-4, and-9)
nsubj(shows-11, it-10)
conj(loses-4, shows-11)
det(blending-15, a-12)
advmod(good-14, very-13)
amod(blending-15, good-14)
dobj(shows-11, blending-15)
prep(blending-15, of-16)
det(figure-19, the-17)
amod(figure-19, human-18)
pobj(of-16, figure-19)
prep(figure-19, with-20)
poss(environment-23, its-21)
amod(environment-23, natural-22)
pobj(with-20, environment-23)
punct(character-28, -LRB--24)
nsubj(character-28, which-25)
cop(character-28, is-26)
det(character-28, a-27)
appos(environment-23, character-28)
prep(character-28, in-29)
pobj(in-29, itself-30)
punct(character-28, -RRB--31)
punct(loses-4, .-32)

Sentence #5 (19 tokens):
The acting is great, and for that we owe credit to the emerging director, Esteban Ramrez.
[Text=The CharacterOffsetBegin=497 CharacterOffsetEnd=500 PartOfSpeech=DT] [Text=acting CharacterOffsetBegin=501 CharacterOffsetEnd=507 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=508 CharacterOffsetEnd=510 PartOfSpeech=VBZ] [Text=great CharacterOffsetBegin=511 CharacterOffsetEnd=516 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=516 CharacterOffsetEnd=517 PartOfSpeech=,] [Text=and CharacterOffsetBegin=518 CharacterOffsetEnd=521 PartOfSpeech=CC] [Text=for CharacterOffsetBegin=522 CharacterOffsetEnd=525 PartOfSpeech=IN] [Text=that CharacterOffsetBegin=526 CharacterOffsetEnd=530 PartOfSpeech=DT] [Text=we CharacterOffsetBegin=531 CharacterOffsetEnd=533 PartOfSpeech=PRP] [Text=owe CharacterOffsetBegin=534 CharacterOffsetEnd=537 PartOfSpeech=VBP] [Text=credit CharacterOffsetBegin=538 CharacterOffsetEnd=544 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=545 CharacterOffsetEnd=547 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=548 CharacterOffsetEnd=551 PartOfSpeech=DT] [Text=emerging CharacterOffsetBegin=552 CharacterOffsetEnd=560 PartOfSpeech=VBG] [Text=director CharacterOffsetBegin=561 CharacterOffsetEnd=569 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=569 CharacterOffsetEnd=570 PartOfSpeech=,] [Text=Esteban CharacterOffsetBegin=571 CharacterOffsetEnd=578 PartOfSpeech=NNP] [Text=Ramrez CharacterOffsetBegin=579 CharacterOffsetEnd=585 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=585 CharacterOffsetEnd=586 PartOfSpeech=.] 
root(ROOT-0, great-4)
det(acting-2, The-1)
nsubj(great-4, acting-2)
cop(great-4, is-3)
punct(great-4, ,-5)
cc(great-4, and-6)
conj(great-4, for-7)
mark(owe-10, that-8)
nsubj(owe-10, we-9)
pcomp(for-7, owe-10)
dobj(owe-10, credit-11)
prep(owe-10, to-12)
det(director-15, the-13)
amod(director-15, emerging-14)
pobj(to-12, director-15)
punct(director-15, ,-16)
nn(Ramrez-18, Esteban-17)
appos(director-15, Ramrez-18)
punct(great-4, .-19)

Sentence #6 (32 tokens):
Furthermore, the characters are very well written and masterfully complex; so, once you become familiarized with them, you can perceive their feelings without them even saying anything.
[Text=Furthermore CharacterOffsetBegin=591 CharacterOffsetEnd=602 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=602 CharacterOffsetEnd=603 PartOfSpeech=,] [Text=the CharacterOffsetBegin=604 CharacterOffsetEnd=607 PartOfSpeech=DT] [Text=characters CharacterOffsetBegin=608 CharacterOffsetEnd=618 PartOfSpeech=NNS] [Text=are CharacterOffsetBegin=619 CharacterOffsetEnd=622 PartOfSpeech=VBP] [Text=very CharacterOffsetBegin=623 CharacterOffsetEnd=627 PartOfSpeech=RB] [Text=well CharacterOffsetBegin=628 CharacterOffsetEnd=632 PartOfSpeech=RB] [Text=written CharacterOffsetBegin=633 CharacterOffsetEnd=640 PartOfSpeech=VBN] [Text=and CharacterOffsetBegin=641 CharacterOffsetEnd=644 PartOfSpeech=CC] [Text=masterfully CharacterOffsetBegin=645 CharacterOffsetEnd=656 PartOfSpeech=RB] [Text=complex CharacterOffsetBegin=657 CharacterOffsetEnd=664 PartOfSpeech=NN] [Text=; CharacterOffsetBegin=664 CharacterOffsetEnd=665 PartOfSpeech=:] [Text=so CharacterOffsetBegin=666 CharacterOffsetEnd=668 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=668 CharacterOffsetEnd=669 PartOfSpeech=,] [Text=once CharacterOffsetBegin=670 CharacterOffsetEnd=674 PartOfSpeech=RB] [Text=you CharacterOffsetBegin=675 CharacterOffsetEnd=678 PartOfSpeech=PRP] [Text=become CharacterOffsetBegin=679 CharacterOffsetEnd=685 PartOfSpeech=VBP] [Text=familiarized CharacterOffsetBegin=686 CharacterOffsetEnd=698 PartOfSpeech=VBN] [Text=with CharacterOffsetBegin=699 CharacterOffsetEnd=703 PartOfSpeech=IN] [Text=them CharacterOffsetBegin=704 CharacterOffsetEnd=708 PartOfSpeech=PRP] [Text=, CharacterOffsetBegin=708 CharacterOffsetEnd=709 PartOfSpeech=,] [Text=you CharacterOffsetBegin=710 CharacterOffsetEnd=713 PartOfSpeech=PRP] [Text=can CharacterOffsetBegin=714 CharacterOffsetEnd=717 PartOfSpeech=MD] [Text=perceive CharacterOffsetBegin=718 CharacterOffsetEnd=726 PartOfSpeech=VB] [Text=their CharacterOffsetBegin=727 CharacterOffsetEnd=732 PartOfSpeech=PRP$] [Text=feelings CharacterOffsetBegin=733 CharacterOffsetEnd=741 PartOfSpeech=NNS] [Text=without CharacterOffsetBegin=742 CharacterOffsetEnd=749 PartOfSpeech=IN] [Text=them CharacterOffsetBegin=750 CharacterOffsetEnd=754 PartOfSpeech=PRP] [Text=even CharacterOffsetBegin=755 CharacterOffsetEnd=759 PartOfSpeech=RB] [Text=saying CharacterOffsetBegin=760 CharacterOffsetEnd=766 PartOfSpeech=VBG] [Text=anything CharacterOffsetBegin=767 CharacterOffsetEnd=775 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=775 CharacterOffsetEnd=776 PartOfSpeech=.] 
root(ROOT-0, written-8)
advmod(written-8, Furthermore-1)
punct(written-8, ,-2)
det(characters-4, the-3)
nsubjpass(written-8, characters-4)
auxpass(written-8, are-5)
advmod(well-7, very-6)
advmod(written-8, well-7)
cc(written-8, and-9)
nn(complex-11, masterfully-10)
conj(written-8, complex-11)
punct(written-8, ;-12)
advmod(perceive-24, so-13)
punct(perceive-24, ,-14)
advmod(familiarized-18, once-15)
nsubj(familiarized-18, you-16)
cop(familiarized-18, become-17)
advcl(perceive-24, familiarized-18)
prep(familiarized-18, with-19)
pobj(with-19, them-20)
punct(perceive-24, ,-21)
nsubj(perceive-24, you-22)
aux(perceive-24, can-23)
parataxis(written-8, perceive-24)
poss(feelings-26, their-25)
dobj(perceive-24, feelings-26)
prep(perceive-24, without-27)
pobj(without-27, them-28)
advmod(saying-30, even-29)
dep(perceive-24, saying-30)
dobj(saying-30, anything-31)
punct(written-8, .-32)

Sentence #7 (49 tokens):
I can't say I liked this picture because of all the hype it generated(in Costa Rica), but rather because of the way it really achieves in showing an interesting portrait of what happens when you interfere with the equilibrium of people's lives.
[Text=I CharacterOffsetBegin=781 CharacterOffsetEnd=782 PartOfSpeech=PRP] [Text=ca CharacterOffsetBegin=783 CharacterOffsetEnd=785 PartOfSpeech=MD] [Text=n't CharacterOffsetBegin=785 CharacterOffsetEnd=788 PartOfSpeech=RB] [Text=say CharacterOffsetBegin=789 CharacterOffsetEnd=792 PartOfSpeech=VB] [Text=I CharacterOffsetBegin=793 CharacterOffsetEnd=794 PartOfSpeech=PRP] [Text=liked CharacterOffsetBegin=795 CharacterOffsetEnd=800 PartOfSpeech=VBD] [Text=this CharacterOffsetBegin=801 CharacterOffsetEnd=805 PartOfSpeech=DT] [Text=picture CharacterOffsetBegin=806 CharacterOffsetEnd=813 PartOfSpeech=NN] [Text=because CharacterOffsetBegin=814 CharacterOffsetEnd=821 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=822 CharacterOffsetEnd=824 PartOfSpeech=IN] [Text=all CharacterOffsetBegin=825 CharacterOffsetEnd=828 PartOfSpeech=PDT] [Text=the CharacterOffsetBegin=829 CharacterOffsetEnd=832 PartOfSpeech=DT] [Text=hype CharacterOffsetBegin=833 CharacterOffsetEnd=837 PartOfSpeech=NN] [Text=it CharacterOffsetBegin=838 CharacterOffsetEnd=840 PartOfSpeech=PRP] [Text=generated CharacterOffsetBegin=841 CharacterOffsetEnd=850 PartOfSpeech=VBD] [Text=-LRB- CharacterOffsetBegin=850 CharacterOffsetEnd=851 PartOfSpeech=-LRB-] [Text=in CharacterOffsetBegin=851 CharacterOffsetEnd=853 PartOfSpeech=IN] [Text=Costa CharacterOffsetBegin=854 CharacterOffsetEnd=859 PartOfSpeech=NNP] [Text=Rica CharacterOffsetBegin=860 CharacterOffsetEnd=864 PartOfSpeech=NNP] [Text=-RRB- CharacterOffsetBegin=864 CharacterOffsetEnd=865 PartOfSpeech=-RRB-] [Text=, CharacterOffsetBegin=865 CharacterOffsetEnd=866 PartOfSpeech=,] [Text=but CharacterOffsetBegin=867 CharacterOffsetEnd=870 PartOfSpeech=CC] [Text=rather CharacterOffsetBegin=871 CharacterOffsetEnd=877 PartOfSpeech=RB] [Text=because CharacterOffsetBegin=878 CharacterOffsetEnd=885 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=886 CharacterOffsetEnd=888 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=889 CharacterOffsetEnd=892 PartOfSpeech=DT] [Text=way CharacterOffsetBegin=893 CharacterOffsetEnd=896 PartOfSpeech=NN] [Text=it CharacterOffsetBegin=897 CharacterOffsetEnd=899 PartOfSpeech=PRP] [Text=really CharacterOffsetBegin=900 CharacterOffsetEnd=906 PartOfSpeech=RB] [Text=achieves CharacterOffsetBegin=907 CharacterOffsetEnd=915 PartOfSpeech=VBZ] [Text=in CharacterOffsetBegin=916 CharacterOffsetEnd=918 PartOfSpeech=IN] [Text=showing CharacterOffsetBegin=919 CharacterOffsetEnd=926 PartOfSpeech=VBG] [Text=an CharacterOffsetBegin=927 CharacterOffsetEnd=929 PartOfSpeech=DT] [Text=interesting CharacterOffsetBegin=930 CharacterOffsetEnd=941 PartOfSpeech=JJ] [Text=portrait CharacterOffsetBegin=942 CharacterOffsetEnd=950 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=951 CharacterOffsetEnd=953 PartOfSpeech=IN] [Text=what CharacterOffsetBegin=954 CharacterOffsetEnd=958 PartOfSpeech=WP] [Text=happens CharacterOffsetBegin=959 CharacterOffsetEnd=966 PartOfSpeech=VBZ] [Text=when CharacterOffsetBegin=967 CharacterOffsetEnd=971 PartOfSpeech=WRB] [Text=you CharacterOffsetBegin=972 CharacterOffsetEnd=975 PartOfSpeech=PRP] [Text=interfere CharacterOffsetBegin=976 CharacterOffsetEnd=985 PartOfSpeech=VBP] [Text=with CharacterOffsetBegin=986 CharacterOffsetEnd=990 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=991 CharacterOffsetEnd=994 PartOfSpeech=DT] [Text=equilibrium CharacterOffsetBegin=995 CharacterOffsetEnd=1006 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=1007 CharacterOffsetEnd=1009 PartOfSpeech=IN] [Text=people CharacterOffsetBegin=1010 CharacterOffsetEnd=1016 PartOfSpeech=NNS] [Text='s CharacterOffsetBegin=1016 CharacterOffsetEnd=1018 PartOfSpeech=POS] [Text=lives CharacterOffsetBegin=1019 CharacterOffsetEnd=1024 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=1024 CharacterOffsetEnd=1025 PartOfSpeech=.] 
root(ROOT-0, say-4)
nsubj(say-4, I-1)
aux(say-4, ca-2)
neg(say-4, n't-3)
nsubj(liked-6, I-5)
ccomp(say-4, liked-6)
det(picture-8, this-7)
dobj(liked-6, picture-8)
mwe(of-10, because-9)
prep(liked-6, of-10)
predet(hype-13, all-11)
det(hype-13, the-12)
pobj(of-10, hype-13)
nsubj(generated-15, it-14)
rcmod(hype-13, generated-15)
punct(in-17, -LRB--16)
prep(generated-15, in-17)
nn(Rica-19, Costa-18)
pobj(in-17, Rica-19)
punct(in-17, -RRB--20)
punct(liked-6, ,-21)
cc(liked-6, but-22)
advmod(of-25, rather-23)
mwe(of-25, because-24)
conj(liked-6, of-25)
det(way-27, the-26)
pobj(of-25, way-27)
nsubj(achieves-30, it-28)
advmod(achieves-30, really-29)
rcmod(way-27, achieves-30)
prep(achieves-30, in-31)
pcomp(in-31, showing-32)
det(portrait-35, an-33)
amod(portrait-35, interesting-34)
dobj(showing-32, portrait-35)
prep(portrait-35, of-36)
nsubj(happens-38, what-37)
pcomp(of-36, happens-38)
advmod(interfere-41, when-39)
nsubj(interfere-41, you-40)
advcl(happens-38, interfere-41)
prep(interfere-41, with-42)
det(equilibrium-44, the-43)
pobj(with-42, equilibrium-44)
prep(equilibrium-44, of-45)
poss(lives-48, people-46)
possessive(people-46, 's-47)
pobj(of-45, lives-48)
punct(say-4, .-49)

Sentence #8 (26 tokens):
Hopefully, Caribe will establish a reference for future projects in Costa Rica, as it is a very good example of nice movie making.
[Text=Hopefully CharacterOffsetBegin=1030 CharacterOffsetEnd=1039 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=1039 CharacterOffsetEnd=1040 PartOfSpeech=,] [Text=Caribe CharacterOffsetBegin=1041 CharacterOffsetEnd=1047 PartOfSpeech=NNP] [Text=will CharacterOffsetBegin=1048 CharacterOffsetEnd=1052 PartOfSpeech=MD] [Text=establish CharacterOffsetBegin=1053 CharacterOffsetEnd=1062 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=1063 CharacterOffsetEnd=1064 PartOfSpeech=DT] [Text=reference CharacterOffsetBegin=1065 CharacterOffsetEnd=1074 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=1075 CharacterOffsetEnd=1078 PartOfSpeech=IN] [Text=future CharacterOffsetBegin=1079 CharacterOffsetEnd=1085 PartOfSpeech=JJ] [Text=projects CharacterOffsetBegin=1086 CharacterOffsetEnd=1094 PartOfSpeech=NNS] [Text=in CharacterOffsetBegin=1095 CharacterOffsetEnd=1097 PartOfSpeech=IN] [Text=Costa CharacterOffsetBegin=1098 CharacterOffsetEnd=1103 PartOfSpeech=NNP] [Text=Rica CharacterOffsetBegin=1104 CharacterOffsetEnd=1108 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=1108 CharacterOffsetEnd=1109 PartOfSpeech=,] [Text=as CharacterOffsetBegin=1110 CharacterOffsetEnd=1112 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=1113 CharacterOffsetEnd=1115 PartOfSpeech=PRP] [Text=is CharacterOffsetBegin=1116 CharacterOffsetEnd=1118 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=1119 CharacterOffsetEnd=1120 PartOfSpeech=DT] [Text=very CharacterOffsetBegin=1121 CharacterOffsetEnd=1125 PartOfSpeech=RB] [Text=good CharacterOffsetBegin=1126 CharacterOffsetEnd=1130 PartOfSpeech=JJ] [Text=example CharacterOffsetBegin=1131 CharacterOffsetEnd=1138 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=1139 CharacterOffsetEnd=1141 PartOfSpeech=IN] [Text=nice CharacterOffsetBegin=1142 CharacterOffsetEnd=1146 PartOfSpeech=JJ] [Text=movie CharacterOffsetBegin=1147 CharacterOffsetEnd=1152 PartOfSpeech=NN] [Text=making CharacterOffsetBegin=1153 CharacterOffsetEnd=1159 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=1159 CharacterOffsetEnd=1160 PartOfSpeech=.] 
root(ROOT-0, establish-5)
advmod(establish-5, Hopefully-1)
punct(establish-5, ,-2)
nsubj(establish-5, Caribe-3)
aux(establish-5, will-4)
det(reference-7, a-6)
dobj(establish-5, reference-7)
prep(reference-7, for-8)
amod(projects-10, future-9)
pobj(for-8, projects-10)
prep(projects-10, in-11)
nn(Rica-13, Costa-12)
pobj(in-11, Rica-13)
punct(establish-5, ,-14)
mark(example-21, as-15)
nsubj(example-21, it-16)
cop(example-21, is-17)
det(example-21, a-18)
advmod(good-20, very-19)
amod(example-21, good-20)
advcl(establish-5, example-21)
prep(example-21, of-22)
amod(making-25, nice-23)
nn(making-25, movie-24)
pobj(of-22, making-25)
punct(establish-5, .-26)

